features/networking/egress-ingress.feature

```diff
@@ -662,3 +662,58 @@ Feature: Egress-ingress related networking scenarios
     When I execute on the pod:
       | curl | --connect-timeout | 5 | --head | www.chsi.com.cn |
     Then the step should fail
+
+
+  # @author jechen@redhat.com
+  # @case_id OCP-44940
+  @admin
+  @4.10 @4.9    
+  @network-ovnkubernetes	
+  Scenario: [bug2000057] No segmentation error occurs in ovnkube-master after egressfirewall resource that references a DNS name is deleted
+    Given the env is using "OVNKubernetes" networkType
+    And I have a project
+    Given I obtain test data file "networking/list_for_pods.json"
+    When I run the :create client command with:
+      | f | list_for_pods.json |
+    Then the step should succeed
+		
+    When I obtain test data file "networking/ovn-egressfirewall/egressfirewall-policy4.yaml"
+    And I run oc create as admin over "egressfirewall-policy4.yaml" replacing paths:
+      | ["metadata"]["namespace"]  | <%= project.name %>   |
+    Then the step should succeed
+
+    And I wait up to 60 seconds for the steps to pass:
+    """
+    When I run the :get admin command with:
+      | resource      | egressfirewall                 |
+      | resource_name | default                        |
+      | o             | jsonpath={.status}             |
+      | n             | <%= project.name %>            |
+    Then the step should succeed
+    And the output should contain:
+      | EgressFirewall Rules applied |
+    """
+
+    When I run the :delete admin command with:
+      | object_type       | egressfirewall           |
+      | object_name_or_id | default                  |
+      | n                 | <%= project.name %>      |
+    Then the step should succeed
+    
+    Given I switch to cluster admin pseudo user
+    Given admin uses the "openshift-ovn-kubernetes" project
+    Given I store the leader node name from the "ovn-kubernetes-master" configmap to the :leadernode clipboard
+    When I run the :get admin command with:
+      | resource      | pod                                 |     
+      | l             | app=ovnkube-master                  |
+      | fieldSelector | spec.nodeName=<%= cb.leadernode %>  |
+      | o             | jsonpath={.items[*].metadata.name}  |
+    Then the step should succeed
+    And evaluation of `@result[:response]` is stored in the :ovnkube_master_podname clipboard
+
+    When I run the :logs client command with:
+      | resource_name | <%= cb.ovnkube_master_podname %> |
+      | c             | ovnkube-master                   |
+      | since         | 30s                              |
+    Then the step should succeed
+    And the output should not contain "panic: runtime error: invalid memory address or nil pointer dereference"
```

Comment:
@jechen0648 
Suggest looking for string "SIGSEGV: segmentation violation code" in the output.

Reply:
@asood-rh   Thanks for your feedback, I have made the change.